common/persistence/dataInterfaces.go

```diff
@@ -549,6 +566,7 @@ type (
 		ReplicationState         *ReplicationState
 		BufferedEvents           []*workflow.HistoryEvent
 		BufferedReplicationTasks map[int64]*BufferedReplicationTask
+		VersionHistories         VersionHistories
```

Comment:
make this a pointer, so we can use a null value for possible backward / forward compatibility issue

Reply:
Done